app/src/test/java/org/simple/clinic/summary/PatientSummaryScreenControllerTest.kt

```diff
@@ -565,8 +566,18 @@ class PatientSummaryScreenControllerTest {
     verify(medicalHistoryRepository).save(eq(updatedMedicalHistory), any())
   }
 
-  private fun randomAnswer(): MedicalHistory.Answer {
-    return MedicalHistory.Answer::class.java.enumConstants.asList().shuffled().first()
+  @Suppress("unused")
+  fun medicalHistoryQuestionsAndAnswers(): List<List<Any>> {
+
```

Comment:
Extra line here?

Most static code analyzers disallow empty lines at the beginning and ending of a function.

Reply:
Whoops, will fix this.

Reply:
Done.